doc/articles/uno-development/Uno-UI-XAML-ResourceTrimming.md

```diff
@@ -12,17 +12,39 @@ In order to determine what is actually used by the application, the Uno Platform
 
 In order to prepare the linking pass:
 - The tooling determines the presence of the `UnoXamlResourcesTrimming` msbuild property
-- During the source generation, the tooling generates a `LinkerHints` class, which contains a set of properties for all `DependencyObject` inheriting classes.
+- During the source generation, the tooling generates a `__LinkerHints` class, which contains a set of properties for all `DependencyObject` inheriting classes.
 - The source generation creates XAML Resources and Bindable Metadata code that conditionally uses those classes behind `LinkerHints` properties.
 - The tooling also embeds an ILLinker substitution file allowing the linker to unconditionally remove the code that conditionally references those properties. For instance, for `LinkerHints.Is_Windows_UI_Xaml_Controls_Border_Available`, any block of `if (LinkerHints.Is_Windows_UI_Xaml_Controls_Border_Available)` will be removed when the `--feature Is_Windows_UI_Xaml_Controls_Border_Available false` parameter is provided to the linker.
```

Comment:
Should we normalize LinkerHints to __LinkerHints in this document ?

Reply:
Yes, I'll make an additional PR for it.